### DISCUSSION TITLE
Training with 2.0 isn't working as well as 1.5

### DISCUSSION THREAD

**Discussion by abirck** [TheLastBen/fast-stable-diffusion#803] [Q&A]
I'm trying to train the model with my face. I trained a SD 1.5 model with about 30 512x512 images and it worked pretty well! I decided to try training the SD 2.0 768x768 model with 50 768x768 images, 5000 steps (writing out a model every 1000 steps).

It trains okay. And then end model kinda works. If you give it prompts with my identifier it generates images that kind of look like me, if you squint a bit. I went through and tried each model from 1000 -> 5000 and it seems like it does kind of learn me better each step? But the end result is still very poor in terms of looking like me.

The SD 1.5 model wasn't perfect but would definitely generate things that you'd go "oh yeah, that's him".

Is the issue SD 2.0? Can I not train on 768x768 images? Something else? Any advice would be great.

**Comment by abirck** [comment]
Doesn't seem like the 768x768 training images were the issue. I trained a v2 512 model with 512 inputs and outputs and same issue.

**Comment by TheLastBen** [comment]
the issue is with the V2 model, the settings need to be tuned, I'm working on it

**Reply by abirck** [reply]
Good to know. Thank you for doing all the hard work for the rest of us!

Do you do 'releases' or anything I can watch to see when the v2 model has reach a reasonable state of tuned?

**Reply by TheLastBen** [reply]
you will find all the updates here : https://github.com/TheLastBen/fast-stable-diffusion/commits/main/fast-DreamBooth.ipynb
